hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestDataSkippingUtils.scala

```diff
@@ -127,8 +127,8 @@ class TestDataSkippingUtils extends HoodieSparkClientTestBase with SparkAdapterS
     val resolvedExpr = resolveExpr(spark, sourceExpr, sourceTableSchema)
     val lookupFilter = DataSkippingUtils.translateIntoColumnStatsIndexFilterExpr(resolvedExpr, indexSchema)
 
-    val spark2 = spark
-    import spark2.implicits._
+    val sparkB = spark
+    import sparkB.implicits._
```

Comment:
```suggestion
    import spark.implicits._
```

Reply:
That doesn't work. It's due to spark being a variable and also an import path thing

Reply:
And the reason I changed it from spark2 to sparkB is because I want to get rid of all references to spark2 (even though in this case it is totally unrelated to spark version 2)

Reply:
OK, this looks a bit weird.  We can keep it for now since it's in test code.